### PR TITLE
init mudblazor changes

### PR DESCRIPTION
**Pull request by ARodrigue7**
also added some default razor pages. just missing proper routing

### CODE REVIEW

**Review by Marcus-Johnson — CHANGES_REQUESTED** [review]
Please delete the pages folder,  the Shared Views are required and looks good

File: NJInsurancePlatform/Views/Shared/MainLayout.razor [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    <MudAppBar Elevation="0">
+        <MudIconButton Icon="@Icons.Material.Filled.Menu" Color="Color.Inherit" Edge="Edge.Start" OnClick="@((e) => DrawerToggle())" />
+        <MudSpacer />
+        <MudIconButton Icon="@Icons.Custom.Brands.MudBlazor" Color="Color.Inherit" Link="https://mudblazor.com/" Target="_blank" />
+        <MudIconButton Icon="@Icons.Custom.Brands.GitHub" Color="Color.Inherit" Link="https://github.com/MudBlazor/MudBlazor/" Target="_blank" />
+    </MudAppBar>
+    <MudDrawer @bind-Open="_drawerOpen" Elevation="1">
+        <MudDrawerHeader>
+            <MudText Typo="Typo.h6">MyApplication</MudText>
+        </MudDrawerHeader>
```
Comment (Marcus-Johnson) [inline NJInsurancePlatform/Views/Shared/MainLayout.razor:17]: Typo="Typo.h6">Insurance Application</MudText>

File: NJInsurancePlatform/Views/Shared/NavMenu.razor [hunk 2]
```diff
@@ -0,0 +1,5 @@
+﻿<MudNavMenu>
+    <MudNavLink Href="" Match="NavLinkMatch.All" Icon="@Icons.Material.Filled.Home">Home</MudNavLink>
```
Comment (Marcus-Johnson) [inline NJInsurancePlatform/Views/Shared/NavMenu.razor:2]: 'Home', 'My Profile' are the only two pages in the navbar for now.

File: NJInsurancePlatform/Views/Shared/_Layout.cshtml [hunk 3]
```diff
@@ -4,9 +4,11 @@
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>@ViewData["Title"] - NJInsurancePlatform</title>
-    <link rel="stylesheet" href="~/lib/bootstrap/dist/css/bootstrap.min.css" />
-    <link rel="stylesheet" href="~/css/site.css" asp-append-version="true" />
+@*    <link rel="stylesheet" href="~/lib/bootstrap/dist/css/bootstrap.min.css" />
```
Comment (Marcus-Johnson) [inline NJInsurancePlatform/Views/Shared/_Layout.cshtml:7]: Delete the entire comment

File: NJInsurancePlatform/_Imports.razor [hunk 4]
```diff
@@ -0,0 +1,11 @@
+﻿@using System.Net.Http
+@using System.Net.Http.Json
+@using Microsoft.AspNetCore.Components.Forms
+@using Microsoft.AspNetCore.Components.Routing
+@using Microsoft.AspNetCore.Components.Web
+@using Microsoft.AspNetCore.Components.Web.Virtualization
+@*@using Microsoft.AspNetCore.Components.WebAssembly.Http*@
```
Comment (Marcus-Johnson) [inline NJInsurancePlatform/_Imports.razor:7]: Delete the comment

**Review by Marcus-Johnson — CHANGES_REQUESTED** [review]

File: NJInsurancePlatform/Views/Shared/_Layout.cshtml [hunk 5]
```diff
@@ -4,8 +4,6 @@
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>@ViewData["Title"] - NJInsurancePlatform</title>
```
Comment (Marcus-Johnson) [inline NJInsurancePlatform/Views/Shared/_Layout.cshtml:6]: This can be simply 'Insurance'

**Review by Marcus-Johnson — CHANGES_REQUESTED** [review]

File: NJInsurancePlatform/Views/Shared/_Layout.cshtml (hunk 5 above)
Comment (ARodrigue7) [reply NJInsurancePlatform/Views/Shared/_Layout.cshtml:6]: committing changes now

**Review by Marcus-Johnson — APPROVED** [review]
Great work!

**Comment by ARodrigue7** [comment]
Thank you sir!

**Pull request merged**